sklearn/decomposition/tests/test_online_lda.py

```diff
@@ -20,23 +19,25 @@
     assert_array_almost_equal,
     if_safe_multiprocessing_with_blas,
 )
+from sklearn.utils.fixes import CSR_CONTAINERS
 
 
-def _build_sparse_mtx():
+def _build_sparse_mtx(csr_container):
```

Comment:
Since in the long term scipy want to deprecate the sparse matrix API in favor of the sparse array API, let's reflect this by updating the name of this helper to be future proof => `_build_sparse_array`.

Reply:
ok sure!